simulations/notebooks/Sampling-subclones.ipynb

```diff
@@ -286,7 +314,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.13"
+   "version": "3.9.10"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "b0fa6594d8f4cbf19f97940f81e996739fb7646882a419484c72d19e05852a7e"
+   }
   }
  },
  "nbformat": 4,
```

Comment:
In the file below, cleaning outputs could also be beneficial. (Github doesn't want to show the content as the diff is too large).